### PR TITLE
Pr.enum group

### PR DESCRIPTION
**Pull request by Person-93**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://travis-ci.org/Person-93/hunter/builds/349724054
  * https://ci.appveyor.com/project/Person-93/hunter/build/1.0.3

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/200
---

### CODE REVIEW

**Comment by Person-93** [comment]
Adding the pull request number seems to have broken the build somehow.
https://ci.appveyor.com/project/Person-93/hunter/build/1.0.4
How could that change have caused this?

File: docs/packages/pkg/EnumGroup.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.. index:: unsorted ; EnumGroup
+
+.. _pkg.EnumGroup:
+
+EnumGroup
+==========
+
+-  `Official <https://github.com/Person-93/EnumGroup>`__
+-  `Hunterized <https://github.com/hunter-packages/EnumGroup>`__
```
Comment (ruslo) [inline docs/packages/pkg/EnumGroup.rst]: There is no hunterized repository, please remove.

File: cmake/configs/default.cmake [hunk 2]
```diff
@@ -393,3 +393,4 @@ hunter_config(zookeeper VERSION 3.4.9-p2)
 hunter_config(tacopie VERSION 2.4.0-h1)
 hunter_config(cpp_redis VERSION 3.5.0-h1)
 hunter_config(IF97 VERSION 2.1.2)
+hunter_config(EnumGroup VERSION 0.0.1)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: "No newline at end of file" warning:
* http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file

File: cmake/projects/EnumGroup/hunter.cmake [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    VERSION
+    0.0.1
+    URL
+    "https://github.com/Person-93/EnumGroup/archive/0.0.1.tar.gz"
+    SHA1
+    e842e9c81865db3006e1b457b5bedea401ae5734
+)
+
+hunter_cmake_args(
+	EnumGroup
```
Comment (ruslo) [inline cmake/projects/EnumGroup/hunter.cmake]: Use 4 spaces for indentation here.

File: docs/packages/pkg/EnumGroup.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    EnumGroup
+
+.. index:: unsorted ; EnumGroup
+
+.. _pkg.EnumGroup:
+
+EnumGroup
+==========
```
Comment (ruslo) [inline docs/packages/pkg/EnumGroup.rst]: Please make it 8 for esthetic reason.

**Comment by ruslo** [comment]
> Adding the pull request number seems to have broken the build somehow

There is no `appveyor.yml` in this commit. You can tweak AppVeyor settings to ignore such jobs.

File: docs/packages/pkg/EnumGroup.rst (hunk 4 above)
Comment (Person-93) [reply docs/packages/pkg/EnumGroup.rst]: It doesn't work with 8. It said the title underline is too short. It works with 9.

File: docs/packages/pkg/EnumGroup.rst (hunk 1 above)
Comment (Person-93) [reply docs/packages/pkg/EnumGroup.rst]: Done

File: cmake/configs/default.cmake (hunk 2 above)
Comment (Person-93) [reply cmake/configs/default.cmake]: Done

File: appveyor.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
+# Windows (https://github.com/travis-ci-tester/toolchain-table)
```
Comment (ruslo) [inline appveyor.yml]: Please remove this file, it doesn't belong here.

File: docs/packages/pkg/EnumGroup.rst (hunk 4 above)
Comment (ruslo) [reply docs/packages/pkg/EnumGroup.rst]: > It doesn't work with 8

Hah, yeah, I failed to count to 9 :) My bad, sorry.

> It said the title underline is too short. It works with 9

Great, thank.

File: cmake/configs/default.cmake (hunk 2 above)
Comment (ruslo) [reply cmake/configs/default.cmake]: Got a conflict here.

Hint: The probability of the conflict will be much lower if you will keep an alphabetical order.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3450
* https://travis-ci.org/ingenue/hunter/builds/353198961?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3452
* https://travis-ci.org/ingenue/hunter/builds/353204809?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.20.19

**Pull request merged**